Makefile

```diff
@@ -51,15 +52,33 @@ _prepare-mobile: ##@prepare Install mobile platform dependencies and prepare wor
 	npm install
 
 $(STATUS_GO_IOS_ARCH):
-	curl -s -L \
+	curl --fail --silent --location \
 		"$(GITHUB_URL)/$(STATUS_GO_VER)/status-go-ios.zip" \
-		-o "$(STATUS_GO_IOS_ARCH)"
+		--output "$(STATUS_GO_IOS_ARCH)"; \
+	if [ $$? -ne 0 ]; then \
+		echo "Failed to download from GitHub, checking DigitalOcean Bucket..."; \
+		curl --fail --silent --location \
+			"${DO_SPACE_URL}/status-go-ios-$(STATUS_GO_VER).zip" \
+			--output "$(STATUS_GO_IOS_ARCH)"; \
+		if [ $$? -ne 0 ]; then \
+			echo "Failed to download from DigitalOcean Spaces!"; \
```

Comment:
Maybe we can add `exit 1` in order to not exit with code `0` in case of failure?